docs/index.rst

```diff
@@ -6,81 +6,89 @@
 Welcome to the OpenFE documentation!
 ====================================
 
+The **OpenFE** toolkit provides a free and open-source framework for calculating alchemical free energies.
```

Comment:
I don't think "alchemical free energies" is something that exists?? But I could be wrong!
Maybe
```suggestion
The **OpenFE** toolkit provides a free and open-source framework for alchemical free energy calculations.
```